editor/src/clj/dynamo/graph.clj

```diff
@@ -83,15 +83,15 @@
    (node-type* (now) node-id))
   ([basis node-id]
    (when-let [n (gt/node-by-id-at basis node-id)]
-     (gt/node-type n basis))))
+     (gt/node-type n))))
 
 (defn node-type
   "Return the node-type given a node. Uses the current basis if not provided."
-  ([node]
-    (node-type (now) node))
-  ([basis node]
-    (when node
-      (gt/node-type node basis))))
+  [node]
+  (gt/node-type node))
```

Comment:
Presumably we should keep the `(when node ...)` clause here to preserve the original behavior? Or are we sure this never gets called with a `nil` node?

Reply:
Good catch, fixed!